cmd/scale.go

```diff
@@ -160,7 +160,7 @@ func (sc *scaleCmd) load() error {
 
 	if sc.containerService.Properties.IsAzureStackCloud() {
 		writeCustomCloudProfile(sc.containerService)
-		err = sc.containerService.Properties.SetAzureStackCloudSpec()
+		err = sc.containerService.Properties.SetAzureStackCloudSpec(false, true)
```

Comment:
nit: rather than pass in boolean literals (e.g., `false, true`) create a named variable so it's more clear what context we're invoking the function in. E.g.:

```
isUpgrade := false
isScale := true
err = sc.containerService.Properties.SetAzureStackCloudSpec(isUpgrade, isScale)
```

(same comment for all invocations of `SetAzureStackCloudSpec` below using the new params interface)